.dockerignore

```diff
@@ -0,0 +1,2 @@
+Dockerfile
```

Comment:
Would be easier to read the diffs without all this docker stuff checked in. Git has a concept of local ignore.